plugin-localization/src/main/java/com/mapbox/mapboxsdk/plugins/localization/LocalizationPlugin.java

```diff
@@ -214,14 +231,15 @@ public void setMapLanguage(@NonNull MapLocale mapLocale) {
     for (Source source : style.getSources()) {
       if (sourceIsFromMapbox(source)) {
         boolean isStreetsV8 = sourceIsStreetsV8(source);
+        boolean isStreetsV7 = sourceIsStreetsV7(source);
```

Comment:
This check can be moved to line 242, otherwise, it's going to be redundant for `streets-v8` sources.

Reply:
I have made above two changes(`name_zh` should be changed to `name_zh_Hans` in streets v8, as they both represent Simplified Chinese). I also changed Locale.CHINA's default MapLocale to MapLocale.CHINESE_HANS, since in streets v8 Simplified Chinese should use `name_zh_Hans`.